src/JSONLD/JSONLD.jl

```diff
@@ -0,0 +1,80 @@
+abstract type JSONLD <: DataRepo
+end
+
+export JSONLD_Web, JSONLD_DOI
+
+include("JSONLD_Web.jl")
+include("JSONLD_DOI.jl")
+
+function description(repo::JSONLD, mainpage)
+    desc = filter_html(handle_keys(mainpage, "description"))
+    authors = get_authors(repo, mainpage)
+    author = format_authors(authors)
+    license = get_license(mainpage)
+    date = get_dates(repo, mainpage)
+
+    """
+    Author: $(author)
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+    """
+end
+
+function get_authors(repo::JSONLD, mainpage)
+    authors = handle_keys(mainpage, "author", "creator")
+    if authors isa Vector
+        return collect(skipmissing(handle_keys.(authors, "name")))
+    elseif authors isa Dict
+        return [handle_keys(authors, "name")]
+    else
+        return []
+    end
+    
+end
+
+function get_dates(repo::JSONLD, mainpage)
+    rawdate = handle_keys(mainpage, "datePublished", "dateCreated", "dateModified")
+    try
+        return Dates.format(Dates.DateTime(rawdate), "U d, yyyy")
+    catch error
+        if error isa MethodError
```

Comment:
is this to handle `rawdate==nothing`
If so better to handle it if an `if rawdate==nothing` than with exception handling.

Exception handling can be the answer to this kind of thing I think.
But much more if there is a sequence of functions that might fail at anypoint,
and you want to treat them all the same.
For just one guarding against it with an `if` is better.
Julia (unlike say `python`) generally prefers avoiding exceptions to handling them.,

Also you forgot the `else rethrow()` in the catch block's `if`

Reply:
This is to handle the difference in the incoming date formats. Some have just the year and some have different formats of date.